pkg/network/ovn_kubernetes_test.go

```diff
@@ -139,6 +141,66 @@ func TestRenderOVNKubernetesIPv6(t *testing.T) {
 	g.Expect(script).To(ContainSubstring("pssl:9641:[::]"))
 }
 
+func TestRenderedOVNKubernetesConfig(t *testing.T) {
+	type testcase struct {
+		desc                string
+		expected            string
+		hybridOverlayConfig *operv1.HybridOverlayConfig
+	}
+	testcases := []testcase{
+		{
+			desc: "default",
+			expected: "[default]\nmtu=\"1500\"\ncluster-subnets=\"10.128.0.0/15/23,10.0.0.0/14/24\"\n" +
+				"encap-port=\"8061\"\n\n[kubernetes]\nservice-cidrs=\"172.30.0.0/16\"\n" +
+				"ovn-config-namespace=\"openshift-ovn-kubernetes\"\napiserver=\"" + fmt.Sprintf("https://%s:%s", os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT")) + "\"\n\n" +
+				"[gateway]\nmode=local\nnodeport=true",
```

Comment:
ick. If you use backticks around the whole thing rather than double-quotes, then you won't have to backslash the internal quotes. Also, you could use actual newlines rather than `\n` in the string to make it more readable, although sometimes that's weird and not worth it because you lose the indentation.

hm, also, I think I'd set `KUBERNETES_SERVICE_HOST` / `KUBERNETES_SERVICE_PORT` at the start of the test case rather than rely on whatever happened to be in the environment. (And then you don't need to `Getenv` and `Sprintf` here, you can just hardcode the values into the expected output.)

Reply:
done